packages/@azure/servicebus/data-plane/test/renewLockSessions.spec.ts

```diff
@@ -26,18 +26,26 @@ describe("Standard", function(): void {
   );
   const namespace = Namespace.createFromConnectionString(SERVICEBUS_CONNECTION_STRING);
 
+  after(async () => {
+    await namespace.close();
+  });
+
   const STANDARD_QUEUE_SESSION =
     process.env.QUEUE_NAME_NO_PARTITION_SESSION || "unpartitioned-queue-sessions";
   describe("Unpartitioned Queue", function(): void {
-    const senderClient = namespace.createQueueClient(STANDARD_QUEUE_SESSION);
-    const receiverClient = senderClient;
+    let senderClient: QueueClient;
+    let receiverClient: QueueClient;
+
     describe("Tests - Lock Renewal for Sessions- Peeklock Mode", function(): void {
       beforeEach(async () => {
+        senderClient = namespace.createQueueClient(STANDARD_QUEUE_SESSION);
+        receiverClient = senderClient;
         await beforeEachTest(receiverClient);
       });
 
       afterEach(async () => {
```

Comment:
Ok, I see the updated diff using afterEach. 
Unclear if these changes are helpful, as error seems to show up in the sample run. If running tests resolve the errors then can ship.

Reply:
Yes, the test failures are indeed resolved with this fix.